hover/sass/effects/border-transitions/_underline.sass

```diff
@@ -0,0 +1,41 @@
+// Underline
+// Use to animate an underline on an element on hover
+
+// Arguments:
+// from = the origin of the underline: left, right, center
+// color = color of the line
+// height = thickness of the line
+// duration = length of time the animation takes
+
+@mixin underline($from, $color, $height, $duration: $mediumDuration)
+    @include hacks()
+    position: relative
+    overflow: hidden
+
+    &::before
+        @include prefixed(transition-property, "left, right")
+        @include prefixed(transition-duration, $duration)
+        @include prefixed(transition-timing-function, ease-out)
+        content: ''
+        position: absolute
+        z-index: -1
+        bottom: 0
+        background: $color
+        height: $height
+
+        @if $from == left
+            right: 100%
+            left: 0
+        @else if $from == right
+            right: 0
+            left: 100%
+        @else
+            right: 50%
+            left: 50%
+
```

Comment:
I'm just concerned that if someone makes an error and puts `@include underline(whatever, ...)`, then next person may not look into `mixins.sass` to see how the mixin actually works, but will copy the mixin from the place where the mistake is and then this mistake will spread. But maybe someone else could express their opinion, @incuna/frontend ?